Swabbr.Core/Notifications/SwabbrNotification.cs

```diff
@@ -41,24 +46,55 @@ public sealed class SwabbrNotification
             UserAgent = "swabbr-backend"; // TODO What to do with this?
         }
 
+        /// <summary>
+        ///     The notification protocol used.
+        /// </summary>
         public string Protocol { get; }
 
+        /// <summary>
+        ///     The version of the protocol used.
+        /// </summary>
         public string ProtocolVersion { get; }
 
+        /// <summary>
+        ///     The type of data sent.
+        /// </summary>
         public string DataType { get; }
 
+        /// <summary>
+        ///     The version of the data sent.
+        /// </summary>
         public string DataTypeVersion { get; }
 
+        /// <summary>
+        ///     The type of notification, <see cref="NotificationAction"/>,
+        ///     as a string value.
+        /// </summary>
         public string ClickAction { get; }
 
+        /// <summary>
+        ///     The type of notification, <see cref="NotificationAction"/>.
+        /// </summary>
+        public NotificationAction ClickActionInt { get; }
```

Comment:
Since the android app currently expects `ClickAction` to be string, I added `ClickActionInt` to make it backwards compatible. This is absolutely open for discussion.

Reply:
Dont see the problem